docs/documentation/cert-manager.md

```diff
@@ -0,0 +1,203 @@
+
+To let Dekorate to generate the certificate and issuer resources, simply declare the following dependency part of your pom file:
+
+```xml
+<dependency>
+  <groupId>io.dekorate</groupId>
+  <artifactId>certmanager-annotations</artifactId>
+  <version>{{site.data.project.release.current-version}}</version>
+</dependency>
+```
+
+And provide the certificate configuration. The minimal information that the Dekorate needs is:
+- `secretName` : the name of the Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) resource that will include the Cert-Manager generated files.
+- the Issuer that represents the certificate authority (CA). See all the supported options in [the Issuer](#issuers) section.
+
+For all the configuration options, please go to [the Configuration guide](https://dekorate.io/configuration-guide/#cert-manager) of the Cert-Manager.
+
+The minimal configuration can be provided using the properties file and the following keys:
+
+```
+dekorate.certificate.secret-name=tls-secret
+# The selfSigned issuer:
+dekorate.certificate.self-signed.enabled=true
+```
+
+Or via the `@Certificate` annotation:
+
+```java
+@Certificate(secretName = "tls-secret", selfSigned = @SelfSigned(enabled = true))
+public class Main {
+    // ...
+}
+```
+
+This configuration will generate up to two resources under `target/classes/dekorate/kubernetes.yml` file that should look like this:
+
+```
+---
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: kubernetes-example
+spec:
+  selfSigned: {}
+---
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: kubernetes-example
+spec:
+  encodeUsagesInRequest: false
+  isCA: false
+  issuerRef:
+    name: kubernetes-example
+  secretName: tls-secret
+```
+
+Apart from these two resources, the Cert-Manager Dekorate extension will also configure, part of the Deployment, a volume mounted from the secret that contain the Cert-Manager generated files to allow the application to access them and to configure the HTTPS/TLS endpoint:
+
+```
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: kubernetes-example
+spec:
+  replicas: 1
+  template:
+    spec:
+      containers:
+        - name: kubernetes-example
+          volumeMounts:
+            - mountPath: /etc/certs
+              name: volume-certs
+              readOnly: true
+      volumes:
+        - name: volume-certs
+          secret:
+            optional: false
+            secretName: tls-secret
+```
+
+#### Usage
+
+To let the application access the path mounted, we have then to update also the properties of the application (Quarkus, Spring Boot, ...) to use the files that are mounted in the folder "/etc/certs" from the secret.
```

Comment:
"To let the application access the path mounted, we have then to update also the properties of the application (Quarkus, Spring Boot, ...) to use the files that are mounted in the folder "/etc/certs" from the secret." 

--> 

"To let the application (Quarkus, Spring Boot, ...) to access the files mounted under the path "/etc/certs" from the secret, we have then to update also the properties of the application to access them."

Maybe add here a snippet example of the properties to set

Reply:
There is a link to the Spirng Boot example in the next line.